### ISSUE TITLE
Hornet w Playbook OOM on CPX41 (16GB)

### ISSUE THREAD

**Issue by centercirclesolutions** [iotaledger/hornet#632]
**Describe the bug**
Hornet was killed due to OOM

**To Reproduce**

**Environment information:**
 - Hetzner CPX41 Ubuntu 20.04 8 Core 16GB
 - Hornet 5.0 (Playbook)

**Additional context**
[dmesg Log](https://ybin.me/p/d136ff2b0ac0e46c#LTG1IrVOjLdVvAD6POCfY+c87m3iKS//BL/GnQqkTjM=)
[Hornet Log](https://github.com/gohornet/hornet/files/5098538/hornet5.OOM.Issue.zip)

**Comment by svenger87** [comment]
Here´s my log
[log.txt](https://github.com/gohornet/hornet/files/5098552/log.txt)

**Comment by TeeVeeEss** [comment]
HORNET version 0.5.0
[hornet_mainnet_crash.log.gz](https://github.com/gohornet/hornet/files/5098768/hornet_mainnet_crash.log.gz)

Standard installation with ``apt``, VM 4 core, 5 GB RAM
OOM lines from ``dmesg``:
```2020-08-19T18:46:01,572195+0200 oom-kill:constraint=CONSTRAINT_NONE,nodemask=(null),cpuset=/,mems_allowed=0,global_oom,task_memcg=/system.slice/hornet.service,task=hornet,pid=13281,uid=998
2020-08-19T18:46:01,572425+0200 Out of memory: Killed process 13281 (hornet) total-vm:30634608kB, anon-rss:4698460kB, file-rss:0kB, shmem-rss:0kB, UID:998 pgtables:51516kB oom_score_adj:0
2020-08-19T18:46:02,827752+0200 oom_reaper: reaped process 13281 (hornet), now anon-rss:0kB, file-rss:0kB, shmem-rss:0kB
```
log created with ``journalctl --no-hostname --utc --since "4 hours ago" -u hornet >/tmp/hornet_mainnet_crash.log``

Panic reason:
``Aug 19 16:46:26 hornet[14000]: panic: total does not match supply: 2779531283277761 != 2779530283277761`` 

Only after deleting db and snapshot I could get HORNET running again.

**Comment by uwec** [comment]
My three nodes (each CPX21, 3 Core, 4 GB RAM, Ubuntu) Hornet 0.5.0 als crashed with OOM. 

[OOM_Node1.txt](https://github.com/gohornet/hornet/files/5099805/OOM_Node1.txt)
[OOM_Node2.txt](https://github.com/gohornet/hornet/files/5099806/OOM_Node2.txt)
[OOM_node3.txt](https://github.com/gohornet/hornet/files/5099807/OOM_node3.txt)

![Node1](https://user-images.githubusercontent.com/46627851/90699781-7681cd00-e284-11ea-8d18-7756437e176d.png)
![Node3](https://user-images.githubusercontent.com/46627851/90699789-797cbd80-e284-11ea-909d-74bdef873c79.png)

**Comment by muXxer** [comment]
Fixed in #637
